pkg/cfn/manager/iam.go

```diff
@@ -31,7 +34,22 @@ func (c *StackCollection) createIAMServiceAccountTask(errs chan error, spec *api
 	}
 	spec.Tags[api.IAMServiceAccountNameTag] = spec.NameString()
 
-	return c.CreateStack(name, stack, spec.Tags, nil, errs)
+	if err := c.CreateStack(name, stack, spec.Tags, nil, errs); err != nil {
+		if !replaceIfExists {
+			return err
```

Comment:
what are the consequences of returning an `AlreadyExists` error here? can `eksctl` not continue on in that case?

Reply:
>can eksctl not continue on in that case?

Yes, if `replaceExistingRole`/`replaceIfExists` isn't set, eksctl should not continue. 

The `replaceExistingRole` variable determines if the caller expects the role may already exist, and to replace it if it does. Right now, only the VPC resource controller needs this operation to be idempotent. For other operations like `create iamserviceaccount`, eksctl should not continue and abort with that error.